torch_xla/csrc/tensor.h

```diff
@@ -225,6 +225,12 @@ class XLATensor {
       double scale, int64_t scatter_dim, int64_t shard_count,
       std::vector<std::vector<int64_t>> groups);
 
+  static ir::Value reduce_scatter_out(XLATensor& output, const XLATensor& input,
```

Comment:
Do you need to follow Google C++ style here? https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs requires you to put output params at the end of the list.